stdlib/scripts/build-stdlib.sh

```diff
@@ -39,4 +39,11 @@ STDLIB_PACKAGE_NAME="stdlib.mojopkg"
 FULL_STDLIB_PACKAGE_PATH="${BUILD_DIR}"/"${STDLIB_PACKAGE_NAME}"
 mojo package "${STDLIB_PATH}" -o "${FULL_STDLIB_PACKAGE_PATH}"
 
+echo "Validating API documentation strings in the Standard Library."
```

Comment:
I don't think you need to modify `stdlib/scripts/build-stdlib.sh`, verifying docstrings is more of something to do once a pull request is done, and a contributor will push just after. It's a separate workflow from building the stdlib or testing it.